app/src/main/java/org/mozilla/focus/widget/FloatingEraseButton.java

```diff
@@ -9,6 +9,9 @@
 import android.support.design.widget.FloatingActionButton;
 import android.util.AttributeSet;
 import android.view.View;
+import android.view.animation.AnimationUtils;
+
+import org.mozilla.focus.R;
 
 public class FloatingEraseButton extends FloatingActionButton {
```

Comment:
Please test this behavior with:

* custom tabs (no fab)
* scrolling (should still disappear/reappear)
* multiple tabs
* switching between foreground/background (with "keep activities" enabled/disabled)

Reply:
I believe it works in all these cases, did you find something that says otherwise?

Reply:
Nope. But I also didn't test all variations myself. :)